src/main/generated/io/vertx/mqtt/MqttClientOptionsConverter.java

```diff
@@ -65,6 +65,8 @@ static void fromJson(Iterable<java.util.Map.Entry<String, Object>> json, MqttCli
             obj.setWillTopic((String)member.getValue());
           }
           break;
+        case "willMessageAsString":
+          break;
```

Comment:
why is this generated and empty here? @vietj ?

Reply:
should be fixed

Reply:
@tsegismont is it normal having that empty one?

Reply:
I think this can be due to the usage of `@GenIgnore`, I made some suggestions to @JeffreyThijs , let's see the result